README.md

```diff
@@ -1,41 +1,384 @@
+* Approval Required from a minimum of 2 people
+
+> [!NOTE]
+> requiring two reviewers will likely slow things down at the benefit of ensuring there's a sufficient amount of visibility on the changes being made. It also prevents bottlenecks from forming. Many people on our team should be able to speak to the features landing in our SDKs
+
+
+## Release Criteria
+* Each Release will have an accompanying Github Release
+* Each Github Release will have an accompanying git tag that matches the version being published
+* For the forseeable future, each SDK is free to publish releases at a frequency that the SDK's respective DRI sees fit
+
+## CI / CD
+Each SDK will use Github Actions for CI/CD
+
+| Feature                       | Typescript | Kotlin | Rust | Swift |
+| ----------------------------- | ---------- | ------ | ---- | ----- |
+| OSS License Check             | ❌          | ❌      | ❌    | ❌     |
+| Running Unit Tests            | ❌          | ❌      | ❌    | ❌     |
+| Publishing Artifacts          | ❌          | ❌      | ❌    | ❌     |
+| Publishing API Reference Docs | ❌          | ❌      | ❌    | ❌     |
+| Example Feature Usage         | ❌          | ❌      | ❌    | ❌     |
+
+## Publishing Artifacts
+Each SDK will be published to the most widely adopted registry/repository for the respective language
+
+
+| SDK        | Repository    | Published (y/n) |
+| ---------- | ------------- | --------------- |
+| Typescript | npm           | ❌               |
+| Kotlin     | maven central | ❌               |
+| Rust       | ?             | ❌               |
+| Swift      | ?             | ❌               |
+
+
+## Publishing API Reference Documentation
+Each SDK will auto generate API reference documentation using the respective language's commenting convention and doc gen tooling 
+
+---
+
+At a _minimum_, API reference documentation will be published to the respective sdk repository's Github Pages. e.g. `https://tbd54566975.github.io/tbdex-kt/`
+
+---
+
+| Language   | Comment Convention | Docs Generator | Published (y/n) |
+| ---------- | ------------------ | -------------- | --------------- |
+| Typescript | TSDoc              | API Extractor  | ✅               |
+| Kotlin     | KDoc               | Dokka          | ✅               |
+| Rust       | ?                  | ?              | ❌               |
+| Swift      | ?                  | ?              | ❌               |
+
+
+> [!IMPORTANT] 
+> Producing API reference documentation is the responsibility of an _implementer_
+
+## Example Feature Usage
+Each SDK will **publish** example usage for _each_ implemented feature. This can either be included as a part of API reference documentation _or_ published separately
+
+# Features
+
+## Cryptographic Digital Signature Algorithms (DSA)
+| Algorithm       | Typescript | Kotlin | Rust | Swift |
+| --------------- | ---------- | ------ | ---- | ----- |
+| `ES256K`        | ✅          | ✅      | ❌    | ❌     |
```

Comment:
I think this is fine to start but would like to see these checks/x's dynamically generated

Reply:
and need to align this with the current test harness

Reply:
that's certainly one of the things that's in the works. We have an HTML report like this now, and from there generating a markdown report shouldn't be too hard. Currently blocked by lack of any functioning tests.

Reply:
agreed. the purpose for adding these tables here manually is to make it easier to figure out what issues need to be created for each respective SDK. the intent is to remove them as mentioned here:

<img width="1116" alt="image" src="https://github.com/TBD54566975/sdk-development/assets/4887440/0222b555-d669-4e30-ae25-d5970069dc35">